controllers/powerprofile_controller_test.go

```diff
@@ -941,9 +1000,7 @@ func TestPowerProfile_Reconcile_SharedFrequencyValuesLessThanAbsoluteValue(t *te
 		}
 
 		_, err = r.Reconcile(context.TODO(), req)
-		if err != nil {
-			t.Errorf("%s failed: expected the reconciler to not have failed", tc.testCase)
-		}
+		assert.ErrorContains(t, err, "maximum or minimum frequency value cannot be")
```

Comment:
error message needs correcting-missing second half of sentence.

Reply:
Same as the other controller errors, will be addressed in the upcoming release. 
I looked into this as the message is coming from the controller as it's parameterized  `fmt.Sprintf("maximum or minimum frequency value cannot be below %d", absoluteMinimumFrequency)`. We need to check in the test where that value is coming from and update the error message